test/ServiceFabric.Tests/Yarp.ReverseProxy.ServiceFabric.Tests.csproj

```
@@ -16,8 +16,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\ReverseProxy.ServiceFabric\Yarp.ReverseProxy.ServiceFabric.csproj" />
-    <ProjectReference Include="..\ReverseProxy.Tests.Common\Yarp.ReverseProxy.Tests.Common.csproj" />
+    <ProjectReference Include="..\..\src\ServiceFabric\Yarp.ReverseProxy.ServiceFabric.csproj" />
+    <ProjectReference Include="..\Tests.Common\Yarp.ReverseProxy.Tests.Common.csproj" />
```

`ReverseProxy` isn't needed in the test common library/namespace anymore is it?
```suggestion
    <ProjectReference Include="..\Tests.Common\Yarp.Tests.Common.csproj" />
```

---

I didn't change project names, only folder.

---

To keep project names consistent with namespaces.

---

I meant to ask if this project and namespace should be renamed since it's global?

---

Ideally, yes. Do you want to do it in this PR?

---

Yes please, it's consistent with the other changes you're making.

---

Done.